src/modules/esl-media-query/test/esl-media-rule-list.test.ts

```diff
@@ -8,7 +8,7 @@ describe('ESLMediaRuleList', () => {
 
   describe('Integration cases:', () => {
     test('Basic case: "1 | @sm => 2 | @md => 3" parsed correctly', () => {
-      const mrl = ESLMediaRuleList.parseQuery('1 | @sm => 2 | @md => 3');
+      const mrl = ESLMediaRuleList.parse('1 | @sm => 2 | @md => 3');
```

Comment:
Pleaes do not change existing tests. We do not change `parseQuery` or `parseTuple` and WWE don't gonna change their coverage.
We should test new parse method according to comments that we define for parseMixed that was here before...